### DISCUSSION TITLE
IPv6 support

### DISCUSSION THREAD

**Discussion by pavanbalaji** [openucx/ucx#6323] [Q&A] [answered]
Hello,

Does UCX have IPv6 support automatically enabled?  I have a few IPv6 devices on my system (and no IPv4, except `lo`), but `ucx_info` does not seem to detect these devices for the "tcp" transport.

Regards,

  -- Pavan

**Comment by shamisp** [comment, marked as answer]
Adding @dmitrygx

**Reply by dmitrygx** [reply]
@pavanbalaji @shamisp currently UCX/TCP doesn't support IPv6.
we could think about adding IPv6 support for the next UCX versions.
